index.html

```diff
@@ -53,6 +54,241 @@ <h1>I'm Redwan.<br/>
                 </div>
             </div>
         </section>
-    </main> 
+        <section class="Projects">
+            <div class="container work-area">
+                <div class="flex-row">
+                    <div class="flex-col">
+                        <img
+                            class="snapshoot-portfolio-icon"
+                            alt=""
+                            src="Images/snapshoot-portfolio@2x.png"
+                        />
+                            <div class="top">
+                            <h2>Tonic</h2>
+                            <div class="client-parent">
+                                <ul>
+                                    <li>CANOPY</li>
+                                    <li>Back End Dev </li>
+                                    <li>2015</li>
+                                </ul>
+                            </div>
+                            <p>
+                                A daily selection of privately <br/>
+                                personalized reads; no accounts or <br/>
+                                sign-ups required.
+                            </p>     
+                            </div>
+                            <div class="tags">
+                                <ul>
+                                    <li>
+                                        <a class="btnlin" href="">HTML</a>
+                                    </li>
+                                    <li>
+                                        <a class="btn" href="">CSS</a>
+                                    </li>
+                                    <li>
+                                        <a class="btn" href="">JavaScript</a>
+                                    </li>
+                                </ul>
+                            </div>
+                            <div class="action">
+                                <button class="submit">See project</button>
+                            </div>
+                    </div>
+                    <div class="flex-col">
+                        <img
+                        class="snapshoot-portfolio-icon"
+                        alt=""
+                        src="Images/Snapshoot Portfolio.png"
+                    />
```

Comment:
- [ ] After fixing the path to your image, I notice that your image is not centered like on the figma template 🎯 🎯 

👀  👇🏽 

![Screenshot from 2023-06-06 01-03-56](https://github.com/ethred/Portfolio/assets/10439283/49cb5261-4429-45d4-a7bc-a397b657bb9f)

You can fix that with this snippet of code 👇🏽 

```sh
.snapshoot-portfolio-icon {
    border-radius: var(--br-5xs);
    width: 100%;
    height: auto;
 }
```